php-checks/src/main/java/org/sonar/php/checks/formatting/FunctionSpacingCheck.java

```diff
@@ -82,22 +82,22 @@ private void checkSpaceForComma(FormattingStandardCheck formattingCheck, AstNode
       Token nextToken = comma.getNextSibling().getToken();
       Token previousToken = comma.getPreviousSibling().getLastToken();
 
-      if (isOnSameLine(previousToken, commaToken, nextToken)) {
-        boolean isSpaceBeforeOK = getNbSpaceBetween(previousToken, commaToken) == 0;
-        boolean isSpaceAfterOK = getNbSpaceBetween(commaToken, nextToken) == 1;
+//      if (isOnSameLine(previousToken, commaToken, nextToken)) {
+//        boolean isSpaceBeforeOK = getNbSpaceBetween(previousToken, commaToken) == 0;
+//        boolean isSpaceAfterOK = getNbSpaceBetween(commaToken, nextToken) == 1;
 
-        if (!isSpaceBeforeOK && isSpaceAfterOK && msgIndex < 0) {
-          msgIndex = 0;
-        } else if (isSpaceBeforeOK && !isSpaceAfterOK && msgIndex < 0) {
-          msgIndex = 1;
-        } else if (!isSpaceBeforeOK && !isSpaceAfterOK) {
-          msgIndex = 2;
-          break;
-        }
-      }
+//        if (!isSpaceBeforeOK && isSpaceAfterOK && msgIndex < 0) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)